deployments/dev/image/binder/Dockerfile

```diff
@@ -1,2 +1,2 @@
 # Note that there must be a tag
-FROM pangeo/pangeo-notebook-onbuild:2019.09.23
+FROM pangeo/pangeo-notebook-onbuild:2019.12.04
```

Comment:
Would it make sense to use the latest tag here? Or is there a downside that I'm not aware of?

Reply:
I think we need to keep pinning versions at this level. For now at least, we need to keep in sync with various versions of dask and dask-gateway between the helm chart, pangeo-stacks, and this repo.